csharp/dotnet/examples/TestApp-2.0/cloudbuild.yaml

```diff
@@ -1,13 +1,18 @@
 # Example build script to create a Docker image from a .NET Core 2.0 Web Api app.
 steps:
 
-# Restore and publish the app. The restore must use a local directory
-# inside of the workspace to preserve the packages across invokations
-# of the dotnet tool.
+# Use a persistent volume to store the restored packages for this app. The
```

Comment:
word choice nit: perhaps "...to **host** the restored packages..." -- "store the restored" sounds redundant to me